recipes-ni/sysconfig-settings/sysconfig-settings.bb

```diff
@@ -0,0 +1,116 @@
+SUMMARY = "System configuration files"
+DESCRIPTION = "Configuration files for the National Instruments System Configuration subsystem."
+HOMEPAGE = "https://github.com/ni/meta-nilrt"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
+
+DEPENDS += "niacctbase base-files-nilrt"
+
+PV = "2.0"
+
+
+
+SRC_URI = "\
+	file://nisetembeddeduixml \
+	file://systemsettings \
+	file://uixml \
+"
```

Comment:
@bstreiff I tried looking around and didn't see any commentary on the original sysconfig-settings PR as to why we were declaring each source file individually. Was there a reason?